ml-algorithms/src/test/java/org/opensearch/ml/engine/algorithms/anomalylocalization/AnomalyLocalizerImplTests.java

```diff
@@ -234,6 +235,15 @@ public void testGetLocalizedResultsForInvalidTimeRange() {
         anomalyLocalizer.getLocalizationResults(input, outputListener);
     }
 
+    @Test(expected = RuntimeException.class)
```

Comment:
Why expect `RuntimeException`  rather `IndexNotFoundException` ?
You can use `ExpectedException` to assert the exception type and error message, refer to https://github.com/opensearch-project/ml-commons/blob/main/common/src/test/java/org/opensearch/ml/common/dataframe/ColumnValueTest.java#L17

Reply:
Yeah was originally checking the specific type of exception in the test, then found out that this test class and the test class you mentioned are all checking `Runtime Exception.class`, so continuing using the same approach. 

Will change it to check specific exception type